reconcile/utils/saasherder/saasherder.py

```diff
@@ -95,11 +96,20 @@ def is_commit_sha(ref: str) -> bool:
     return bool(re.search(r"^[0-9a-f]{40}$", ref))
 
 
-RtRef = tuple[str, str, str]
 Resource = dict[str, Any]
 Resources = list[Resource]
 
 
+@dataclass
+class PromotionRef:
+    """Utility class to identify a target in a saas file.
+    Used to make promotion validations"""
+
+    saas_file: SaasFile
+    resource_template: SaasResourceTemplate
+    resource_template_target: SaasResourceTemplateTarget
+
+
```

Comment:
Please move this dataclass to `models.py` 🙇

Reply:
done